packages/react/src/app-layout/AppLayoutSidebarDialog.tsx

```diff
@@ -74,6 +74,7 @@ export function AppLayoutSidebarDialog({
 							<AnimatedBox
 								display={{ [APP_LAYOUT_DESKTOP_BREAKPOINT]: 'none' }}
 								role="dialog"
+								aria-label="Menu"
```

Comment:
I originally planned to use `aria-labelledby` to the Menu button span, however, VoiceOver didn't announce that and also Storybook would still fail